app/services/doc_auth_client.rb

```diff
@@ -0,0 +1,28 @@
+class DocAuthClient
+  def self.client
+    @doc_auth_client ||= begin
```

Comment:
Do we need a `.reset!` method so we don't pollute tests accidentally? Also are all these clients fairly stateless? Is there a cost benefit to memoizing?

Reply:
Yeah, they are. No benefit. I think this is a result of copy pasting this logic. I'll put this out.